analyses/stackanalyses/controller.go

```diff
@@ -195,3 +198,52 @@ func (mc *Controller) getManifestName(manifestFile string) string {
 	}
 	return stats.Name()
 }
+
+// ProcessSummary processes summary results, return true if Vul found
+func ProcessSummary(analysedResult driver.GetResponseType, jsonOut bool) bool {
```

Comment:
Move summary related functionalities to summary pkg

Reply:
Ack.